profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/tmgmt_poetry_mock.behat.inc

```diff
@@ -99,7 +99,7 @@ class TMGMTPoetryMockSubContext extends RawDrupalContext implements DrupalSubCon
   /**
    * Remove pending translations in a scenario tagged with @poetry.
    *
-   * @AfterScenario @poetry
+   * @poetry
```

Comment:
By removing '@AfterScenario' part this function will not be called at the end of the '@poetry' tagged scenarios.